### PR TITLE
add hardcoded values to define files

### PR DESCRIPTION
**Pull request by Bechir-Brahem**

### CODE REVIEW

File: pyctbgui/services/SlowADCs.py [hunk 1]
```diff
@@ -10,13 +12,13 @@ def setup_ui(self):
         pass
 
     def connect_ui(self):
-        for i in range(8):
+        for i in range(Defines.numSlowAdc):
```
Comment (thattil) [inline pyctbgui/services/SlowADCs.py:15]: Why dont you do the same way you did for signals and dacs.. I had liked that.. you could do "slowadc.count"?

File: pyctbgui/services/SlowADCs.py [hunk 2]
```diff
@@ -10,13 +12,13 @@ def setup_ui(self):
...
             getattr(self.mainWindow, f"pushButtonSlowAdc{i}").clicked.connect(partial(self.updateSlowAdc, i))
         self.mainWindow.pushButtonTemp.clicked.connect(self.updateTemperature)
 
     def refresh(self):
         self.updateSlowAdcNames()
-        for i in range(8):
+        for i in range(Defines.numSlowAdc):
```
Comment (thattil) [inline pyctbgui/services/SlowADCs.py:21]: same here?

**Review by thattil — CHANGES_REQUESTED** [review]
LGTM! Just those super minor changes for numslowadc. :)

File: pyctbgui/services/SlowADCs.py (hunk 1 above)
Comment (Bechir-Brahem) [reply pyctbgui/services/SlowADCs.py:15]: Okay I will do that

**Review by thattil — APPROVED** [review]

**Pull request merged**